1.9/overview/architecture/boot-sequence.md

```diff
@@ -0,0 +1,44 @@
+---
+post_title: Boot Sequence
+nav_title: Boot Sequence
+menu_order: 6
+---
+
+During DC/OS installation, the components come online in a particular sequence due to interdependencies.
+
+## Master nodes
+
+On each master node the following happens, in chronological order:
```

Comment:
not a big fan of using "here" in docs this way... It's also not an "installation sequence" so much as an initialization/boot/startup sequence, and not every step as described is a new component...